compiler/next/lib/types/QualifiedType.cpp

```diff
@@ -67,23 +67,25 @@ static const char* kindToString(QualifiedType::Kind kind) {
   return "unknown";
 }
 
-std::string QualifiedType::toString() const {
+void QualifiedType::stringify(std::ostream& ss,
+                              chpl::StringifyKind stringKind) const {
   const char* kindStr = kindToString(kind_);
-  std::string typeStr = (type_)?(type_->toString()):(std::string("nullptr"));
+  std::ostringstream strstream;
+  type_->stringify(strstream, stringKind);
+  // TODO: What is this typeStr doing? it doesn't get used?
+  std::string typeStr = (type_)?(strstream.str()):(std::string("nullptr"));
```

Comment:
yeah I think you should just delete this line as well as the `type_->stringify` line above. it is handled below.